test/integration/fixtures/blocks/core__separator-color.json

```diff
@@ -4,8 +4,8 @@
 		"isValid": true,
 		"attributes": {
 			"opacity": "alpha-channel",
-			"backgroundColor": "accent",
-			"tagName": "hr"
+			"tagName": "hr",
+			"backgroundColor": "accent"
```

Comment:
This is a fixture for the spacer block, so I don't think any changes are necessary.